### PR TITLE
Add filtering

### PR DESCRIPTION
**Pull request by peter-mouland**
* [x] Apply to comparator data
 * [x] Device Filtering
 * [x] Cohort Filtering
 * [x] Region Filtering
 * [x] Referrer Filtering
 * [x] Turn on in prod

### CODE REVIEW

File: src/server/queries/Articles.js [hunk 1]
```diff
@@ -136,19 +147,19 @@ export default function PageViewsQuery(query) {
...
+        "terms" : {
+          "field" : "geo_region"
         }
       },
-      countries: {
-        terms : {
-          field : 'geo_country'
+      "countries": {
+        "terms" : {
+          "field" : "geo_country"
```
Comment (radiodario) [inline src/server/queries/Articles.js]: we don't need \" where we're going

**Comment by radiodario** [comment]
It's not working for me, i only see region and referrers, and when i try to filter i get really strange behaviours:

screen shot 2015-10-07 at 16 35 23
screen shot 2015-10-07 at 16 35 07

**Comment by peter-mouland** [comment]
imgres

**Comment by radiodario** [comment]
merging

**Comment by peter-mouland** [comment]
will, let you know when i've applied the filter to comparators

File: src/shared/lib/daterangepicker.css [hunk 2]
```diff
@@ -143,7 +143,6 @@
 .daterangepicker .calendar th, .daterangepicker .calendar td {
   white-space: nowrap;
   text-align: center;
-  min-width: 32px;
```
Comment (peter-mouland) [inline src/shared/lib/daterangepicker.css]: firefox fix

**Comment by radiodario** [comment]
cool, merge it!

**Pull request merged**